RIRImageResizing/Classes/NSObject/RIRResizeImageOperation.h

```diff
@@ -0,0 +1,33 @@
+//
+//  RIRResizeImageOperation.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "RIRResizeImageOperationParameters.h"
+
+
+
+
+
+@interface RIRResizeImageOperation : NSObject
+
+#pragma mark - resizeParameters
+@property (nonatomic, strong, nullable) RIRResizeImageOperationParameters* resizeParameters;
```

Comment:
All these properties need to be readonly on the outside. The redeclare them internally without readonly (as they are now) so you can call the setter methods internally.